plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -159,6 +172,12 @@
     private static final String OPENSHIFT_VOLUME_STORAGE_CLASS           = "volume.beta.kubernetes.io/storage-class";
     private static final String OPENSHIFT_VOLUME_STORAGE_CLASS_NAME      = "che-workspace";
     private static final String OPENSHIFT_IMAGE_PULL_POLICY_IFNOTPRESENT = "IfNotPresent";
+    private static final String IDLING_ALPHA_OPENSHIFT_IO_IDLED_AT       = "idling.alpha.openshift.io/idled-at";
+    private static final String IDLING_ALPHA_OPENSHIFT_IO_PREVIOUS_SCALE = "idling.alpha.openshift.io/previous-scale";
+    private static final String OPENSHIFT_CHE_SERVER_DEPLOYMENT_NAME     = "che";
+    private static final String OPENSHIFT_CHE_SERVER_SERVICE_NAME        = "che-host";
```

Comment:
We need the deployment config and service of the che server (che and che-host), not of a workspace. How can I get them?